lib/vdc/BOVCollection.cpp

```diff
@@ -217,21 +242,37 @@ int BOVCollection::_validateParsedValues()
         _brickSizeAssigned = true;
     }
 
-    // Validate endian type
-    if (_tmpDataEndian != _dataEndian && _dataEndianAssigned == true)
-        return _inconsistentValueError(ENDIAN_TOKEN);
-    else {
-        _dataEndian = _tmpDataEndian;
-        _dataEndianAssigned = true;
-    }
-
     // Validate byte offest
     if (_tmpByteOffset != _byteOffset && _byteOffsetAssigned == true)
         return _inconsistentValueError(OFFSET_TOKEN);
     else {
         _byteOffset = _tmpByteOffset;
         _byteOffsetAssigned = true;
     }
+
+    // If _dataFile is not an absolute path, prepend with the BOV header's path
+    if (!Wasp::FileUtils::IsPathAbsolute(_dataFile)) { _dataFile = _currentFilePath + "//" + _dataFile; }
```

Comment:
Does it really make sense do generate the path to the data file here? The name of this function is 'validateParsedValues'. One would expect the function to simply validate the inputs. Instead, as a side effect this function is modifying _dataFile. Why not simply handle the pathing issue in the Initialize loop where the rest of the file  path handling code resides?

Reply:
I can do that.  

I was thinking that validating something can mean to check if its correct, or to make something valid.  But since this is the only variable I'm correcting, I'll put it in the Initialize() function instead.

Reply:
`/` is not a valid file separator on Windows. `FileUtils::JoinPaths` will do it correctly.

Reply:
@stasj I'm 90% sure `//` is, but this is moot thanks to the FileUtils function.